apps/docs/src/app/pages/component-pages/dialog-page/dialog-page.component.ts

```diff
@@ -0,0 +1,24 @@
+import { DialogService } from '@anglify/components';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { DialogTestComponent } from './dialog-test.component';
+
+@Component({
+  selector: 'app-dialog-page',
+  templateUrl: './dialog-page.component.html',
+  styleUrls: ['./dialog-page.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DialogPageComponent {
+  public constructor(private readonly dialogService: DialogService) {}
+
+  public openModal() {
+    this.dialogService.open(DialogTestComponent).subscribe({
```

Comment:
A `no-unsafe-member-access` error is displayed in this line. Please fix/rework this.

Reply:
Oh, never mind. The IDE was only slow here. That is not a problem anymore